### ISSUE TITLE
Plugin documentation badges

### ISSUE THREAD

**Issue by sagikazarmark** [thephpleague/tactician#27]
I would rather use badges for links than the current solution.

I mean this:

```
Composer | [league/:package_name](https://packagist.org/packages/league/:package_name)
Github | [thephpleague/:package_name](https://github.com/thephpleague/:package_name)
Authors | [:author_name](http://twitter.com/:twitter_name)
```

Actually, I would use the same badges as on the main page.

**Comment by rosstuck** [comment]
Tried it originally, didn't find it super readable but it's cool with me for bling factor.

If you want to add the build status ones, I'd suggest keeping the format and just adding one more row called "Build" to the table and putting multiple badges in there (though I think we can't break those into new lines which was one of the reasons I stopped doing it in the first place? Don't remember). Keeps it compact but skimmable at a glance.
